internal/api/migrate/mapping.cuepart

```diff
@@ -165,7 +165,12 @@ spec: {
                 if grafanaVar.sort != _|_ if #mapping.sort[grafanaVar.sort] != _|_ {
                     sort: #mapping.sort[grafanaVar.sort]
                 }
-                // defaultValue: nothing to map to this field
+                if grafanaVar.current != _|_ if grafanaVar.current.value != _|_ {
```

Comment:
```suggestion
                if (*(grafanaVar.current.value) | _|_ ) != _|_ {
```
reusing your below trick. I guess that should work here too?

Reply:
I guess :)